src/rest/identities/client.ts

```diff
@@ -19,6 +21,17 @@ export class Identities {
     return this.client.post(`/authorizations/${id}/accept`, { ...params });
   }
 
+  public async getPendingAuthorizations(
+    id: string,
+    type: AuthorizationType
```

Comment:
```suggestion
    type?: AuthorizationType
```

Is this supposed to be optional? Otherwise seems like no point in having the if in the body